### PR TITLE
Improvement - icon page main template

### PR DESCRIPTION
**Pull request by helloimela**
**Describe pull-request**  
Update icon page templae

**Solving issue**  
Part of
Fixes: #92 

**How to test**  
1. Setup local strapi on this branch: https://github.com/scania/corporate-ui-cms/pull/16
2. Update database
3. Run corporate-ui-site with localhost (fetchData.js is pointing to localhost for now)
4. Check Icon Page following basic layout from Design
5. Check all pages is working with the new setup

**Screenshots**  
Current view in Icon Page
image

**Additional context**  
In XD, it has line above icon-list, will be added in another PR as part of Icon List task.

### CODE REVIEW

**Comment by helloimela** [comment]
Note: Before merge, I need to change back fetchData pointing to fetch from SDDS instead

**Review by mmexvr — CHANGES_REQUESTED** [review]
image
Color of the icon is incorrect

Might have been fixed another branch?

**Comment by helloimela** [comment]
> image
> Color of the icon is incorrect
> 
> Might have been fixed another branch?

The color in the design is #161743 but we don't have it in our theme, so I make it the same with other icons (Scania blue). Since we will get a revamp on colour & foundation, should we keep it like this for now?

File: src/app/components/icon-list/icon-list.component.ts [hunk 1]
```diff
@@ -9,23 +9,22 @@ import { Theme } from '../../app.interface';
...
-  }
+    // remove whitespaces
+    this.content=this.content.trim();
 
-  ngOnDestroy() {
-    this.subscribeStore.unsubscribe();
+    // assign icons into array
+    this.icons = this.content.split('\n');
+    // icons[0]: "category=xyz" - remove category from array
+    this.icons.shift();
```
Comment (mmexvr) [inline src/app/components/icon-list/icon-list.component.ts:27]: Can be removed

Comment (helloimela) [reply src/app/components/icon-list/icon-list.component.ts:27]: This PR is still uses data from iconList component in cms, so it still need to parse a block of list separated with \n.

**Review by mmexvr — APPROVED** [review]

**Pull request merged**